linera-base/src/vm.rs

```diff
@@ -65,3 +67,32 @@ pub enum EvmQuery {
     /// A request to schedule an operation that can mutate the application state.
     Mutation(Vec<u8>),
 }
+
+/// An EVM mutation containing a value and argument data
+#[derive(Default, Serialize, Deserialize)]
+pub struct EvmMutation {
+    /// The amount being transferred
+    pub value: alloy_primitives::U256,
+    /// The encoded argument data
+    pub argument: Vec<u8>,
+}
+
+/// Creates an internal mutation from value and argument data
+pub fn get_evm_mutation(
```

Comment:
Can we just add a constructor `new` and write this?
`bcs::to_bytes(&EvmOperation::new(value, argument))`